src/Profiler/DataCollector/FixtureLoaderCollector.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Profiler\DataCollector;
+
+use Neusta\Pimcore\FixtureBundle\Profiler\FixtureReference\FixtureReference;
+use Neusta\Pimcore\FixtureBundle\Profiler\FixtureReference\FixtureReferenceResolver;
+use Neusta\Pimcore\FixtureBundle\Profiler\PerformanceInfo\PerformanceInfo;
+use Neusta\Pimcore\FixtureBundle\Profiler\Timing\TimingCollector;
+use Symfony\Bundle\FrameworkBundle\DataCollector\AbstractDataCollector;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class FixtureLoaderCollector extends AbstractDataCollector
```

Comment:
What do you think about making everything `final` and `@internal`? This is no public API, right?

```suggestion
/**
 * @internal
 */
final class FixtureLoaderCollector extends AbstractDataCollector
```